packages/backend/src/peripherals/coingecko/CoingeckoQueryService.ts

```diff
@@ -29,33 +26,81 @@ export class CoingeckoQueryService {
     to: UnixTime,
     granularity: Granularity,
     address?: EthereumAddress,
-  ): Promise<PriceHistoryPoint[]> {
+  ): Promise<QueryResultPoint[]> {
+    const [start, end] = adjustAndOffset(from, to, granularity)
+
+    const prices = (
+      await this.queryCoinMarketChartRange(
+        coingeckoId,
+        start,
+        end,
+        granularity,
+        address,
+      )
+    ).prices
```

Comment:
why the braces `()`

Reply:
they are needed for the await expression